lib/driver/runtimeTool.cpp

```diff
@@ -101,7 +101,22 @@ ToolOnModule(WasmEdge::VM::VM &VM, const std::string &FuncName,
       case TypeCode::V128:
         fmt::print("{}\n"sv, uint128((*Result)[I].first.get<uint128_t>()));
         break;
-      /// TODO: FuncRef and ExternRef
+      case TypeCode::Ref: {
+        auto HTCode = (*Result)[I].second.getHeapTypeCode();
```

Comment:
I think it's propper to use `isFuncRefType()` and `isExternRefType()` APIs to check the ref cases.
Be ware of the other cases. You can print `<anyref>` for the other cases.